netbox/utilities/templatetags/helpers.py

```diff
@@ -234,10 +234,19 @@ def utilization_graph(utilization, warning_threshold=75, danger_threshold=90):
     """
     Display a horizontal bar graph indicating a percentage of utilization.
     """
+    if isinstance(utilization, int):
+        return {
+            'utilization': utilization,
+            'warning_threshold': warning_threshold,
+            'danger_threshold': danger_threshold,
+        }
+
     return {
-        'utilization': utilization,
+        'utilization': utilization[0],
         'warning_threshold': warning_threshold,
         'danger_threshold': danger_threshold,
+        'util_count': utilization[1],
```

Comment:
Bit of a nit, but I'd prefer to use more verbose names for these. `util_count` isn't immediately clear as to its purpose (we often use "util" for "utility"). Maybe `utilization_count` and `utilization_total`?